code/datums/components/harvester.dm

```diff
@@ -30,18 +30,16 @@
 	var/datum/action/harvester/reagent_select/reagent_select_action
 	///The maximum amount that one chemical can be loaded
 	var/max_loadable_reagent_amount = 30
-	var/loadup_on_attack = FALSE
+	var/loadup_on_attack = TRUE
```

Comment:
Just delete this entirely and bake it in to default behaviour

Reply:
done, don't know why i didn't do this in the first place